### PR TITLE
Add support for different encoding in ANT neuro CNT format reader

### PR DESCRIPTION
**Pull request by mscheltienne**
Depends on https://github.com/conda-forge/antio-feedstock/pull/6#issuecomment-2547387329

### CODE REVIEW

**Comment by mscheltienne** [comment]
@larsoner It would be nice to include this in 1.9, but I don't know yet why the `conda-forge` release failed to build on arm64..

**Comment by larsoner** [comment]
Yes we can get it in! But you'll have to pay the price of letting me push a couple of unrelated commits to your PR :see_no_evil:

File: mne/io/ant/ant.py [hunk 1]
```diff
@@ -102,7 +105,7 @@ def __init__(
                 "Missing optional dependency 'antio'. Use pip or conda to install "
                 "'antio'."
             )
-        check_version("antio", "0.3.0")
+        check_version("antio", "0.5.0")
```
[the remark this answers is not in the document]
Comment (larsoner) [reply mne/io/ant/ant.py:108]: Yes good call

[the remark this answers is not in the document]
Comment (mscheltienne) [reply mne/io/ant/ant.py:108]: Yes, thanks for the catch!

File: mne/io/ant/ant.py [hunk 2]
```diff
@@ -93,16 +94,12 @@ def __init__(
...
         verbose=None,
     ) -> None:
         logger.info("Reading ANT file %s", fname)
-        if importlib.util.find_spec("antio") is None:
-            raise ImportError(
-                "Missing optional dependency 'antio'. Use pip or conda to install "
-                "'antio'."
-            )
-        check_version("antio", "0.3.0")
+        _soft_import("antio", "reading ANT files", min_version="0.5.0")
```
Comment (larsoner) [inline mne/io/ant/ant.py:102]: Realized this `check_version` was a noop -- it returns a `bool`, doesn't raise an error. So I just improved `_soft_import` to have a `min_version` arg which is really the goal here I think.

**Comment by larsoner** [comment]
Thanks @mscheltienne !

**Pull request merged**